tests/FakeItEasy.Tests/Expressions/CallExpressionParserTests.cs

```diff
@@ -28,7 +28,7 @@ public CallExpressionParserTests()
         public void Should_return_parsed_expression_with_instance_method_set()
         {
             // Arrange
-            var call = Call(() => string.Empty.Equals(null, StringComparison.CurrentCulture));
+            var call = Call(() => string.Empty.Equals(null, StringComparison.Ordinal));

```

Comment:
compiler preferred. It doesn't really matter, since we're just using it to specify the signature…